src/Common/CurrentMetrics.cpp

```diff
@@ -5,6 +5,9 @@
 #define APPLY_FOR_METRICS(M) \
     M(Query, "Number of executing queries") \
     M(Merge, "Number of executing background merges") \
+    M(Parts, "Total number of data parts") \
+    M(PartsActive, "Number of active data parts") \
+    M(PartsInactive, "Number of inactive data parts") \
```

Comment:
I was recently thought about these metrics and maybe it worth track separate counters for wide/compact/memory parts? (since limits for number of in-memory parts should be different)

Reply:
Yes, it will be useful. BTW, you can create another PR that will include this PR.

Reply:
I will take a look at this later. Since I want to rethink some code around, in particular, `max_partitions_per_insert_block` for in-memory data parts 100 does not looks like good threshold